stats/view/worker.go

```diff
@@ -104,6 +87,24 @@ func (v *View) Subscribe() error {
 	return <-req.err
 }
 
+// Subscribe begins collecting data for a view. Once a view is subscribed, it reports data
+// via the exporters.
+// Once the view is no longer needed to export data,
+// you may call Unsubscribe to stop collecting data for the view.
+func Subscribe(views ...*View) error {
+	var errstr []string
+	for _, v := range views {
+		err := v.Subscribe()
+		if err != nil {
+			errstr = append(errstr, fmt.Sprintf("%s: %v", v.name, err))
+		}
+	}
+	if len(errstr) > 0 {
```

Comment:
This is programmatically impossible to handle.

One of the main reason why we have the subscription model is to enable/disable collection in prod.

How can we programmatically handle subscription errors?